index.tt

```diff
@@ -6,152 +6,353 @@
 
 <link rel="alternate" type="application/rss+xml" title="RSS" href="/news-rss.xml" />
 
-<div class="nixos-org-index">
-  [%# Title for the site, used to keep the document tree rooted at h1. %]
-  <h1>NixOS</h1>
-  
-  <div class="jumbotron-two"><div>
-    <div class="intro-nix">
-      <h2>What is Nix?</h2>
-    
-      <p>
-      Nix is a powerful package manager for Linux and other Unix systems
-      that makes package management reliable and reproducible.  It
-      provides atomic upgrades and rollbacks, side-by-side installation
-      of multiple versions of a package, multi-user package management
-      and easy setup of build environments.
-      </p>
-    
-      <div class="learn-more">
-        <a class="btn btn-success" href="[%root%]nix/">
-          <i class="fa fa-chevron-right"></i> Learn more about Nix
-        </a>
+<div class="row-fluid">
+
+  <div class="span6">
+    <h1>Reproducible builds and deployments.</h1>
+    <p class="lead">
+      NixOS package manager can be used independently on any POSIX compliant
+      operating system to reliably share your development and build
+      environments across machines.
+    </p>
+    <p class="lead">
+      NixOS, an independently developed GNU/Linux distribution, aims to improve
+      the state of the art in system configuration management. The entire
+      operating system, including the kernel, applications, system packages and
+      configuration files, are built from a single location.
```

Comment:
```suggestion
      configuration files, are built from a single configuration.
```

Reply:
"a single location" / "a single configuration" is not very clear. (What does "single" mean? It's thousands of files...) Maybe something like "a reproducible specification".

Reply:
>What does "single" mean? It's thousands of files...

a single file that is used to build NixOS (`What does "single" mean? It's thousands of files...`). it can import content from other files like in other programming languages to organize big configurations and make it more modular, but that is something for advanced users of NixOS. the landingpage is for newcomers

i think "configuration" is the right word here. it's what admins work with. that's why it's called "configuration management". it's also `configuration.nix` and not "NixOS specification"

as a former admin, i don't really know how a "specification" would look like. but i have some experiences with configuration files and some expectations how they look

Reply:
I reverted Nix and NixOS description the current description that is present in nixos.org. we can comeback to this and improve it in separate PRs.